modules/manage/partials/authentication.adoc

```diff
@@ -770,7 +758,7 @@ Following are additional validation claims (JWT properties) that are included in
 * `exp`: Expiration. The timestamp listed is greater than current time. Must validate within acceptable bounds of the value specified in `oidc_clock_skew_tolerance`. A clock skew tolerance period may be configured by an Admin to account for clock drift between Redpanda and the OIDC Identity Provider (IdP).
 * `iss`: Issuer. Must exactly match the `issuer` property of the JSON returned from the URL specified in `oidc_discovery_url`.
 * `scope`: Scope. Must include the value `openid`.
-* `sub`: Subject. Must match exactly the Principal ID found through the principal mapping.
+* `sub`: Subject. This default claim identifies the principal subject. In Redpanda, it can be mapped to a recognized Redpanda principal for secure authentication and authorization.
```

Comment:
Any claim can be mapped to a redpanda principal, this is just the default, "$.sub".